static/templates/dashboard/dashboard.html

```diff
@@ -1,5 +1,5 @@
 
-<md-data-table class="table">
+<div layout="row"><div flex="60" ng-if="true"><md-data-table class="table">
```

Comment:
div layout as row to allow sidenav to be stacked to the right.
ngIf was for testing. Can be omitted completely.